natro_macro.ahk

```diff
@@ -16182,29 +16182,29 @@ nm_PolarQuestProg(){
 		;determine quest bar sizes and spacing
 		if(QuestBarGapSize=0 || QuestBarSize=0 || QuestBarInset=0) {
 			Loop, 3 {
-				xi := 0
-				yi := PolarStart[3]+15
-				ww := 306
-				wh := PolarStart[3]+100
-				ImageSearch, FoundX, FoundY, windowX + xi, windowY + yi, windowX + ww, windowY + wh, *5 nm_image_assets\questbargap.png
+				xi := windowX
+				yi := windowY+PolarStart[3]+15
+				ww := windowX+306
+				wh := windowY+PolarStart[3]+100
+				ImageSearch, FoundX, FoundY, xi, yi, ww, wh, *5 nm_image_assets\questbargap.png
 				if(ErrorLevel=0) {
-					QuestBarSize:=FoundY-PolarStart[3]
+					QuestBarSize:=FoundY-windowY-PolarStart[3]
```

Comment:
This line (and equivalent duplicates for other questgivers) was not updated to CoordMode Screen, causing the bug.